runtime/doc/lsp.txt

```diff
@@ -406,6 +406,10 @@ LspSignatureActiveParameter
     Used to highlight the active parameter in the signature help. See
     |vim.lsp.handlers.signature_help()|.
 
+                                                           *hl-SnippetTabstop*
+SnippetTabstop
+    Used to color tabstops in snippets.
```

Comment:
Snippets are not really LSP specific (they live under `vim.snippet`, after all), so I don't think they should be documented here.

Reply:
Snippets specifically are designed to follow LSP specification and are added (only) to provide its full coverage. I understand that they are implemented in separate module, yet in spirit they are closer to `hl-Lsp***` highlight groups than `:h highlight-groups` (which do not contain any highlight groups coming from built-in Lua modules).

Reply:
Yes, because the LSP specification has become a lingua franca for snippets. They are not _intended_ to be LSP-only (even if that's all they're _currently_ used for _in core_). I'm afraid I'll have to insist.